app/presenters/tree_node/middleware_server.rb

```diff
@@ -0,0 +1,5 @@
+module TreeNode
+  class MiddlewareServer < Node
+    set_attribute(:title) { |object| object.name.presence + '-' + object.host_name }
```

Comment:
So it is convention or it is not effective? :-) I'm ok with convention, but if it is not effective, i would like to see why, i will look at it.